pymchelper/utils/runmc.py

```diff
@@ -50,6 +50,7 @@ def main(args=None):
     import pymchelper
 
     parser = argparse.ArgumentParser()
+    parser.add_argument('-t', '--type', help='Simulator type: shieldhit, topas or fluka (default: shieldhit)', type=str, default='shieldhit')
```

Comment:
There is another option with the same name:
![image](https://github.com/DataMedSci/pymchelper/assets/7374733/11d5890c-32a1-44f2-83a6-4daec4699f95)


Also - use an Enum for selecting simulator, the same way as `OutputDataType`